### PR TITLE
tweak some templates to ensure displaying efficiently

### PR DESCRIPTION
**Pull request by jon571**

### CODE REVIEW

File: src/app/pages/quickMaps/components/quickMapsHeader.component/quickMapsHeader.component.scss [hunk 1]
```diff
@@ -34,7 +34,7 @@
   margin-left: 2%;
 }
 
-.qmTitle {
+.qm-title {
```
Comment (Fuhji) [inline src/app/pages/quickMaps/components/quickMapsHeader.component/quickMapsHeader.component.scss:37]: case conventions! 
image

File: src/app/pages/quickMaps/components/quickMapsHeader.component/quickMapsHeader.component.ts [hunk 2]
```diff
@@ -11,17 +11,16 @@ import { MicronutrientMeasureType } from 'src/app/apiAndObjects/objects/enums/mi
...
-  public ROUTES = AppRoutes;
+  public readonly ROUTES = AppRoutes;
+  public readonly MEASURE_TYPES = MicronutrientMeasureType;
+
   constructor(
     public route: ActivatedRoute,
     public quickMapsService: QuickMapsService,
     private sharingService: SharingService,
-  ) {}
+  ) { }
```
Comment (Fuhji) [inline src/app/pages/quickMaps/components/quickMapsHeader.component/quickMapsHeader.component.ts:21]: what is causing this space? prettier rules do not add this so must be something local to your vscode setup?

**Review by Fuhji — COMMENTED** [review]
1. class naming conventions, starting to use a single standard now or one-off change?
2. Your VScode is formatting differently to the project settings, would be good to identify what the cause is

File: src/app/pages/quickMaps/components/quickMapsHeader.component/quickMapsHeader.component.ts (hunk 2 above)
Comment (jon571) [reply src/app/pages/quickMaps/components/quickMapsHeader.component/quickMapsHeader.component.ts:21]: I'll have a look what I've got enabled.  Probably a vscode plugin...

File: src/app/pages/quickMaps/components/quickMapsHeader.component/quickMapsHeader.component.scss (hunk 1 above)
Comment (jon571) [reply src/app/pages/quickMaps/components/quickMapsHeader.component/quickMapsHeader.component.scss:37]: In EPOS sonarQube was used to do an evaluation of the GUI.  This kinda thing came up loads.  I think kabab-case is standard in css.  Probably one of the more standard standards!

File: src/app/pages/quickMaps/components/quickMapsHeader.component/quickMapsHeader.component.ts (hunk 2 above)
Comment (Fuhji) [reply src/app/pages/quickMaps/components/quickMapsHeader.component/quickMapsHeader.component.ts:21]: VSCode output tab will let you see which extension is doing the modify on save
image

**Comment by jon571** [comment]
conflict with develop resolved

**Review by Fuhji — APPROVED** [review]

**Pull request merged**